src/node_worker.cc

```diff
@@ -428,6 +475,8 @@ void Worker::JoinThread() {
 Worker::~Worker() {
   Mutex::ScopedLock lock(mutex_);
 
+  active_worker_thread_count--;
```

Comment:
nit: There’s no need to lock variables not protected by the mutex (and I think at this point the lock could be removed here entirely but I’m not 100 % sure without having tried it myself)

```suggestion
  active_worker_thread_count--;
  
  Mutex::ScopedLock lock(mutex_);
```